### PR TITLE
3.91.1

### PR DESCRIPTION
**Pull request by phansys**
### To Do:
- [x] Wait for #6915 in order to ship that fix.

### CODE REVIEW

**Review by VincentLanglet — DISMISSED** [review]

**Comment by phansys** [comment]
Tests are failing because symfony/symfony#40383.

**Comment by axzx** [comment]
in composer.json
```
{
    "conflict": {
        "psr/container": "^1.1"
    }
}
```

**Comment by phansys** [comment]
Sure @axzx, thank you.
#6915.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.91.1](https://github.com/sonata-project/SonataAdminBundle/compare/3.91.0...3.91.1) - 2021-03-05
```
Comment (VincentLanglet) [inline CHANGELOG.md:5]: Don't forget to update the date, when we'll decide if #6915 is needed

**Comment by core23** [comment]
I closed #6915. You can release the patch release @VincentLanglet

**Review by VincentLanglet — APPROVED** [review]

**Review by core23 — APPROVED** [review]

**Pull request merged**